test/integration/parachain/release/redeem.test.ts

```diff
@@ -94,35 +113,35 @@ describe("redeem", () => {
         });
     }).timeout(18 * 60000);
 
-    it("should cancel a redeem request", async () => {
-        await runWhileMiningBTCBlocks(bitcoinCoreClient, async () => {
-            const issueAmount = InterBtcAmount.from.BTC(0.001);
-            const redeemAmount = InterBtcAmount.from.BTC(0.0009);
-            const initialRedeemPeriod = await redeemAPI.getRedeemPeriod();
-            await sudo(redeemAPI, (api) => api.setRedeemPeriod(0));
-            const [, redeemRequest] = await issueAndRedeem(api, electrsAPI, btcRelayAPI, userBitcoinCoreClient, userAccount, vaultToBan.address, issueAmount, redeemAmount, false, ExecuteRedeem.False);
-    
-            // Wait for redeem expiry callback
-            await new Promise<void>((resolve, _) => {
-                redeemAPI.subscribeToRedeemExpiry(newAccountId(api, userAccount.address), (requestId) => {
-                    if (stripHexPrefix(redeemRequest.id.toString()) === stripHexPrefix(requestId.toString())) {
-                        resolve();
-                    }
-                });
-            });
-            await redeemAPI.cancel(redeemRequest.id.toString(), true);
-            const redeemRequestAfterCancellation = await redeemAPI.getRequestById(redeemRequest.id);
-            assert.isTrue(redeemRequestAfterCancellation.status === RedeemStatus.Reimbursed, "Failed to cancel issue request");
-            // Set issue period back to its initial value to minimize side effects.
-            await sudo(redeemAPI, (api) => api.setRedeemPeriod(initialRedeemPeriod));
-        });
-    }).timeout(5 * 60 * 1000);
+    // TODO: Uncomment after `subscribeToRedeemExpiry` is reimplemented
+    // it("should cancel a redeem request", async () => {
```

Comment:
Could you skip this instead of commenting out? That way it should show up in the test runner as a skipped test which would make it visible (and more likely to be remembered!).